src/Renderer/c3DEngine.js

```diff
@@ -546,6 +555,49 @@ define('Renderer/c3DEngine', [
 
     };
 
+    var  unpack1K  = function (color,factor) {
+
+            var bitSh = new THREE.Vector4( 1.0/( 256.0 * 256.0 * 256.0 ),1.0/( 256.0 * 256.0 ), 1.0/256.0, 1.0 );
+            return bitSh.dot(color) * factor;
+    }
+
+
+    /**
+     *
+     * @param {Vecto2D} mouse : mouse position on screen in pixel
+     * @returns {int} uuid's node
+     * */
+    c3DEngine.prototype.getPickingId = function(mouse) {
```

Comment:
`getXXX` naming scheme is generally used for getter function.
Suggestions: `screenCoordsToId` ? `screenCoordsToMeshId`?